apiary.apib

```diff
@@ -6543,6 +6543,24 @@ Available Statuses:
 
 ## List Jobs [/v2/storage/jobs]
 ### Jobs list [GET]
+
++ Parameters
+    + limit (optional, number) - Pagination limit
+        + Default: 100
+    + offset (optional, number) - Pagination offset
+        + Default: 0
+    + storage_job_status (optional) - filter only jobs with status [waiting, processing, error, success]
+    + excluded_storage_job_status (optional) - filter only jobs without status [waiting, processing, error, success]
+    + created_time (optional) - filter jobs created in exact time
+    + created_time_from (optional) - filter jobs created after time
+    + created_time_to (optional) - filter jobs created before time
+    + start_time (optional) - filter jobs started in exact time
+    + start_time_from (optional) - filter jobs started after time
+    + start_time_to (optional) - filter jobs started before time
+    + end_time (optional) - filter jobs end in exact time
+    + end_time_from (optional) - filter jobs end after time
+    + end_time_to (optional) - filter jobs end before time
```

Comment:
Tady by bylo potřeba dopsat které filtry se kterýma nefungujů, protože to bude zmatečné.
když nastavíš start_time tak nefunguje start_time_from a start_time_to
storage_job_status má přednost před excluded_storage_job_status apod,…

Reply:
good point, jdu to tam napsat

Reply:
lepší?